WorkplaceByFacebook/index/MembersInfoConfig.json

```diff
@@ -0,0 +1,59 @@
+{
+  "Services": [
+    {
+      "Url": "https://graph.facebook.com/",
+      "Paging": {
+        "PageSize": 1000,
+        "Parameters": {
+          "Limit": "limit"
+        },
+        "OffsetType": "url",
+        "NextPageKey": "paging.next"
+      },
+      "Endpoints": [
+        {
+          "Method": "GET",
+          "Path": "/community/members",
+          "Headers": {
+            "Authorization": "Bearer @ApiKey"
+          },
+          "QueryParameters": {
+            "fields": "id,name,email,title,locale,organization,division,department,picture,updated_time,start_date,about,active,managers,reports,groups,badges",
+            "inactive": "1"
+            },
+          "ItemPath": "data",
+          "ItemType": "WorplaceMember",
+          "Uri": "https://test2307.workplace.com/profile.php?id=%[id]",
+          "ClickableUri": "https://test2307.workplace.com/profile.php?id=%[id]",
```

Comment:
Same as other comment below

Reply:
It should be solutions788 like the others.